src/main/java/uk/gov/hmcts/juror/api/moj/domain/authentication/UserDetailsDto.java

```diff
@@ -49,5 +53,9 @@ public UserDetailsDto(User user, List<UserCourtDto> courts) {
         this.userType = user.getUserType();
         this.roles = user.getRoles();
         this.courts = courts;
+
+        if (SecurityUtil.isAdministration() || SecurityUtil.isManager() && SecurityUtil.isCourt()) {
```

Comment:
Would be good to have this as its own method like a private boolean canEditApprovalLimit()

Reply:
updated